promenade/templates/common/etc/dnsmasq.d/upstream-dns

```diff
@@ -0,0 +1,9 @@
+no-hosts
+no-resolv
+
+strict-order
+
+server={{ config['Network']['cluster_dns'] }}
```

Comment:
They will not.  Instead, they need to be able to resolve `kubernetes` to reach the cluster api servers to join.